catalog/gitops/configsync/README.md

```diff
@@ -0,0 +1,36 @@
+# ConfigSync package
+
+Configure Config Sync to use Cloud Source Repository (CSR).
+
+## Setters
+
+```
+Setter           Usages
+cluster-name     8
+configsync-dir   1
+deployment-repo  1
+namespace        3
+project-id       9
+```
+
+## Sub-packages
+
+
+## Resources
+
+```
+File                    APIVersion                         Kind               Name                                        Namespace
+config-management.yaml  configmanagement.gke.io/v1         ConfigManagement   config-management                           
+configsync-iam.yaml     iam.cnrm.cloud.google.com/v1beta1  IAMPolicyMember    source-reader-sync-cluster-name-project-id  config-control
+configsync-iam.yaml     iam.cnrm.cloud.google.com/v1beta1  IAMPolicyMember    sync-cluster-name                           config-control
+configsync-iam.yaml     iam.cnrm.cloud.google.com/v1beta1  IAMServiceAccount  sync-cluster-name                           config-control
+setters.yaml            v1                                 ConfigMap          setters
```

Comment:
This won't/shouldn't be applied

Reply:
The script to generate this doesn't ignore local-config=true files. Should it?

Also, most of the other setter.yaml files in this branch do not have the local-config=true annotation. So they are actually applied...

Reply:
I'm putting the script in a different PR.

Reply:
[kptv1 will filter out fnconfig](https://github.com/GoogleContainerTools/kpt/blob/2a817f60cf7132c88fd2e526c02b800cf927c048/pkg/live/load_test.go#L70-L92) defined in the kptfile if applied via `kpt live apply`

Reply:
Yeah, kpt filters out the function configs. Function configs should not be listed with actual resources.